ts/utils/payment.ts

```diff
@@ -307,3 +317,21 @@ export const getPaymentOutcomeCodeDescription = (
   }
   return none;
 };
+
+export const getBancomatOrCreditCardPickMethodDescription = (
```

Comment:
```suggestion
export const getPickPaymentMethodDescription = (
```
You could avoid to include the type of card accepted by the function in its name, the type 

```
 CreditCardPaymentMethod
    | PrivativePaymentMethod
    | BancomatPaymentMethod
```
talks for it

Reply:
Modified in [4761e18](https://github.com/pagopa/io-app/pull/2926/commits/4761e1882f01ff94d6b090c8e99c8a4348c49e43)